src/components/modules/blog/cta/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { StyledBlogCTASubtitle, StyledBlogCTAWrapper } from './cta.styles';
+import type { BlogCTAProps } from './cta.types';
+
+import { Button } from '@/components/elements/button';
+import { Heading } from '@/components/elements/heading';
+import { CaretRight } from '@/components/elements/icon';
+
+export const BlogCTA = ({ title, buttonText, subtitle, buttonHref }: BlogCTAProps) => {
```

Comment:
@RBoraurito, There is a similar box in the [FAQ](https://www.figma.com/file/O5411bxKHP2O2OyfW19w6E/HTTP-Toolkit---UI-Design?type=design&node-id=410-14582&mode=design&t=HvFdqTpp01XaWWod-0) pages, Could we name this to a more generic name to reuse it in the two places? Something like BoxCTA or similar:

![Screenshot 2024-02-28 at 19 49 12](https://github.com/bejamas/httptoolkit-website/assets/22168189/497288e2-9580-4078-9ccd-7c21fa9f1d31)

Reply:
![image](https://github.com/bejamas/httptoolkit-website/assets/58434305/76d9d7f6-9cce-4569-a57f-abaca92e209f)
![image](https://github.com/bejamas/httptoolkit-website/assets/58434305/4d65d493-3848-4cdb-a3b3-14c391f8c194)

Reply:
I just add the icon to the blog cta example